src/vs/workbench/contrib/welcome/telemetryOptOut/browser/telemetryOptOut.ts

```diff
@@ -139,7 +139,6 @@ export abstract class AbstractTelemetryOptOut implements IWorkbenchContribution
 						run: () => {
 							logTelemetry(true);
 							this.configurationService.updateValue('telemetry.enableTelemetry', false);
-							this.configurationService.updateValue('telemetry.enableCrashReporter', false);
```

Comment:
@deepak1556 we should write to `argv.json` in this case when the user wants to opt out